samples/README.md

```diff
@@ -1,13 +1,93 @@
-## Description
-These are the samples for skyflow-python. In order to use these samples, you must first:
+# PYTHON-SDK sample templates
+Use this folder to test the functionalities of PYTHON-SDK just by adding `VAULT-ID` `VAULT-URL` and `SERVICE-ACCOUNT` details at the required place.
 
-1. Install skyflow module using pip
-2. Replace placeholders like `<YOU_VAULT_ID>`, `<YOUR_VAULT_URL>` etc. in the sample file with your Vault ID, Vault URL etc.
+## Prerequisites
+- A Skylow account. If you don't have one, you can register for one on the [Try Skyflow](https://skyflow.com/try-skyflow) page.
+- Python 3.7.0 and above.
 
-Below is a brief description of the samples:
+## Configure
+- Before you can run the sample app, create a vault
+- The package can be installed using pip:
 
-- [SATokenSample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/SATokenSample.py): Contains a python program illustrating the usage of `service_account.generate_bearer_token()`
-- [InsertSample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/InsertSample.py) : Contains a python program illustrating the use of `vault.Client.insert()`
-- [detokenizeSample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/detokenizeSample.py): Contains a python program illustrating the use of `vault.detokenize()`
-- [`getByIDSample.py`](https://github.com/skyflowapi/skyflow-python/blob/main/samples/getByIDSample.py): Contains a python program illustrating the use of `vault.Client.get_by_id()`
-- [`invoke_connection.py`](https://github.com/skyflowapi/skyflow-python/blob/main/samples/invoke_connectionSample.py): Contains a python program illustrating the use of `vault.Client.invoke_connection()`
\ No newline at end of file
+        pip install skyflow
+
+### Create the vault
+1. In a browser, navigate to Skyflow Studio and log in.
+2. Create a vault by clicking **Create Vault** > **Start With a Template** > **Quickstart vault**.
+3. Once the vault is created, click the gear icon and select **Edit Vault** Details.
+4. Note your Vault URL and Vault ID values, then click Cancel. You'll need these later.
+
+### Create a service account
+1. In the side navigation click, **IAM** > **Service Accounts** > **New Service Account**.
+2. For Name, enter **Test-Python-Sdk-Sample**. For Roles, choose the required roles for specific action.
+3. Click **Create**. Your browser downloads a **credentials.json** file. Keep this file secure, as you'll need it in the next steps.
```

Comment:
"3. Click **Create**. Your browser downloads a **credentials.json** file. Keep this file secure. You'll need it for each of the samples."